submissions/onyinyeChukwuka/css/productpage.css

```diff
@@ -0,0 +1,346 @@
+body {
+    font-family: 'Montserrat', 'Raleway', sans-serif;
+    background-color: #ffffff;
+}
+
+/* for the header */
+
+.header {
+    background-color: #0066f5;
+}
+
+nav {
+    padding: 20px;
+}
+
+nav>.right-side>div {
+    margin-right: 100px;
+    font-size: 16px;
+    color: #ffffff;
+    line-height: 1.19;
+    text-align: left;
+}
+
+.tab {
+    color: #ffffff;
+    text-decoration: none;
+}
+
+.link-wrapper {
+    height: 22px;
+}
+
+.link-wrapper a {
+    color: #ffffff;
+    text-decoration: none;
+    font-weight: normal;
+}
+
+.camidescription h3, h4 {
+    font-size: 25px;
+    font-weight: bold;
+    line-height: 1.2;
+    text-align: left;
+    color: #5f5f5f;
+}
+
+/* for the description */
+
+.description h4 {
+    font-size: 22px;
+    font-weight: bold;
+    line-height: 1.23;
+    text-align: left;
+    color: #5f5f5f;
+}
+
+.description h3 {
+    font-size: 20px;
+    line-height: 1.25;
+    text-align: left;
+    color: #5f5f5f;
+}
+
+#button {
+    width: 174px;
+    height: 64px;
+    border-radius: 20px;
+    background-color: #f79521;
+    font-size: 22px;
+    font-weight: bold;
+    line-height: 1.23;
+    text-align: center;
+    color: #ffffff;
+}
+
+/* for the related products */
+
+.related {
+    background-color: #ebebeb;
+    padding-bottom: 40px;
+}
+
+.related h4 {
+    font-family: Montserrat;
+    font-size: 25px;
+    font-weight: bold;
+    line-height: 1.2;
+    text-align: center;
+    color: #5f5f5f;
+    padding-top: 20px;
+}
+
+.display {
+    margin-top: 20px;
+}
+
+.cami {
+    width: 80%;
+    border: 1px solid #707070;
+    margin: auto
+}
+
+img {
+    width: 100%;
+    object-fit: contain;
+}
+
+.martens {
+    background-color: #ffffff;
+}
+
+.martens, .platforms, .military {
+    width: 85%;
+    margin: auto;
+}
+
+.martens h4 {
+    font-size: 22px;
+    font-weight: bold;
+    line-height: 1.23;
+    text-align: center;
+    color: #5f5f5f;
+}
+
+.platforms {
+    background-color: #ffffff;
+}
+
+.platforms h4 {
+    font-size: 22px;
+    font-weight: bold;
+    line-height: 1.23;
+    text-align: center;
+    color: #5f5f5f;
+}
+
+.military {
+    background-color: #ffffff;
+}
+
+.military h4 {
+    font-size: 22px;
+    font-weight: bold;
+    line-height: 1.23;
+    text-align: center;
+    color: #5f5f5f;
+}
+
+/* for the footer */
+
+.footer {
+    background-color: #0066f5;
+    padding: 20px;
+    color: #fcf8f8;
+}
+
+.facebook{
+    width: 45px;
+    height: 41px;
+}
```

Comment:
Your formatting from 160-172 is not consistent with what you have on previous lines.